js/classes/entities/midground/weapon/slingshot.js

```diff
@@ -105,6 +110,7 @@ class Slingshot {
         }
     }
 
+    //TODO remove this when we have a Chad animation containing the slingshot
     draw() {
         if (!this.isHidden) {
```

Comment:
When you get time the slingshot sprite work is done which we can draw in Chad and outsource function to this class I am thinking.

Reply:
Sounds good. I'll get on that next.